scripts/core/superdesk-api.d.ts

```diff
@@ -790,10 +790,15 @@ declare module 'superdesk-api' {
 
     // PAGE
 
+    /**
+     * When not enabled, existing monitoring hiding functionality has to work.
+     */
+    export type IFullWidthPageCapabilityConfiguration = {enabled: false} | {enabled: true; allowed: false} | {enabled: true; allowed: true; onToggle: () => void};
```

Comment:
what's the difference between enabled and allowed? might be also useful to get the full width state in the `onToggle` callback so the component knows how it changed

Reply:
I've clarified the difference in the interface comments